arangodb-net-standard/CollectionApi/PostCollectionResponse.cs

```diff
@@ -1,10 +1,12 @@
-﻿namespace ArangoDBNetStandard.CollectionApi
+﻿using ArangoDBNetStandard.Enums;
+
+namespace ArangoDBNetStandard.CollectionApi
 {
     public class PostCollectionResponse
     {
         public bool Error { get; set; }
 
-        public int Code { get; set; }
+        public HTTPCodes Code { get; set; }
```

Comment:
I think we should create a separate issue to make the change more visible, if we want to change that.